obal/data/playbooks/lint.yml

```diff
@@ -9,6 +9,10 @@
       command: "rpmlint {{ inventory_dir }}/{{ package_base_dir }}{{ inventory_hostname }}"
       register: output
 
+    - name: "Check for a new changelog entry"
+      rpmspec_changelog_check:
+        directory: "{{ inventory_dir }}/{{ package_base_dir }}{{ inventory_hostname }}"
```

Comment:
This does not require changing unless you feel it makes sense. A general comment that for something like a `rpmspec_changelog_check` I'd expect to pass it a spec file not a directory containing a spec file since this acts upon a specfile. Or potentially one or the other.

Reply:
I was mostly sticking with the way that `rpmlint` is working immediately above -- taking a directory rather than a path to a specfile.